plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -115,7 +115,23 @@ const fetchPluginData = async ({createNode, reporter}) => {
                         type: 'JenkinsPlugin',
                         contentDigest: crypto.createHash('md5').update(`plugin${pluginData.name.trim()}`).digest('hex')
                     }
-                }).then(() => {
+                });
+                if (!p || !p.then) {
+                    console.log('p', p, plugin);
```

Comment:
is this supposed to still be here? debug?

Reply:
Hmmmm.. in theory it should always return a promise but sometimes it doesn't. So wanted to track it down.

Thinking about it now maybe I should sentry it. Next pr will be doing a mass upgrade of all deps so hopefully it'll fix it but I guess sentry is easier to debug than VPN locked ci

Reply:
Its going to sentry now